pkg/scheduler/plugins/tdm/tdm.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2021 The Volcano Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tdm
+
+import (
+	"fmt"
+
+	"k8s.io/klog"
+	"k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+
+	"volcano.sh/volcano/pkg/scheduler/api"
+	"volcano.sh/volcano/pkg/scheduler/framework"
+)
+
+const (
+	// PluginName indicates name of volcano scheduler plugin.
+	PluginName = "tdm"
+	// PreemptableTaskKey pod annotation key
+	PreemptableTaskKey = "volcano.sh/preemptable"
```

Comment:
prefer to move them to global const, so other plugins may also reuse this one :)

Reply:
Moved it into `pkg/scheduler/api/well-known-labels. go`